### DISCUSSION TITLE
use two machine, one as farmer one as plotter

### DISCUSSION THREAD

**Discussion by knightblackm** [Chia-Network/chia-blockchain#5629] [Q&A]
if i didn't use same wallet, is possible to use a machine as farmer and machine as plotter?solo and pool farming!

**Comment by aa2001** [comment]
For plotting, you can use anything, via the command line:
.\chia.exe plots create -k 32 -b 3389 -u 128 -r 2 -t D:\TEM -d E:\PLOTS -x -n 1 -f "Hex farmer public key" - p "Hex public key of pool"

For see "Hex public key of pool" and  "Hex farmer public key" use:
.\chia keys show

For HELP use:
.\chia -h

Pool farming, configure by instruction from pool, chia not used for farming, only plots.
If you farm from one key to Solo and Pool at once, you will get a block.
